packages/datadog-plugin-mocha/src/index.js

```diff
@@ -121,6 +130,22 @@ function createWrapRunTests (tracer, testEnvironmentMetadata, sourceRoot) {
   }
 }
 
+const nameToParams = {}
```

Comment:
the instrumentation in `mocha-each` needs to carry information for the `mocha` instrumentation. Do we do this anywhere else? Should I carry this in the `tracer` itself?